conf/resources/data/etmp.eligibility.SIMP/hasInvalidInterestSignalsFalse.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
can this be renamed to "rlsSignalValueFalse.json as it looks like its validating "RLS"